### ISSUE TITLE
Using in with expo to web

### ISSUE THREAD

**Issue by adrielcardoso** [react-native-maps/react-native-maps#2966]
# Question
how to render in platform web it?

**Comment by NickDiVona** [comment]
I am fairly certain that this library doesn't support react-native web yet. I hope it does eventually, but it makes sense why it doesn't out of the box.

**Comment by vrinch** [comment]
> I am fairly certain that this library doesn't support react-native web yet. I hope it does eventually, but it makes sense why it doesn't out of the box.

Any updates on this?

**Comment by Agent215** [comment]
Yeah this would be a great feature to have. Im not sure what would need to be done to make this happen, anyone have any thoughts on where to start?

**Comment by NickDiVona** [comment]
So I still love the idea of there being an out-of-the-box solution for React-Native Web but am confused as to how it would work. 

Apple provides free Apple Maps API use for iOS devices, and Google provides free Google Maps API use for Android devices. 

A decent amount of people choose to use Google Maps on both platforms just for the assurance that things will look and behave the same across mobile devices, however, it is my understanding that you do get billed for your Google Maps usage on iOS devices as you need an API key to get that working. As I understand it, you cannot use Apple Maps on Android so more devs either use the native implementation, or they use Google Maps across both. It could be done under-the-hood to take in the same Google Maps API key that you'd use for your iOS client with relative ease, as all of the data is handled the same. 

To summarize: 
I believe configuring RN-Web with Google Maps as a default option is best as Google Maps already supports iOS very well. 

Anyone else have any ideas or suggestions?
